### DISCUSSION TITLE
Is there any file structure or documentation on creating your own dataset?

### DISCUSSION THREAD

**Discussion by JustAHippo** [coqui-ai/TTS#1172] [General Q&A] [answered]
I was looking to train coqui-ai on my own custom dataset of a specific person, and I wasn't sure how to compile a dataset of .wav files into a dataset compatible with coqui-ai. Is there a way to do that?

**Comment by Ca-ressemble-a-du-fake** [comment]
If you want to create your own custom dataset you can mimic the file structure of LJSpeech datasets.
Basically the file structure to follow is like (based on https://stackoverflow.com/q/51123147/17524305):

main folder:

    -wavs (folder must be called "wavs")

        -001.wav

        -002.wav

        -etc (you can choose the filenames as you wish)
    -metadata.csv: this is a CSV file where "|" is used as column separator. It gathers the transcript of each wav file. So for coqui-ai each line has to be like (without double-quotes) : "001|transcript of the file wavs/001.wav"

Mind that the "wavs" directory and the wav extension are omitted on purpose in metadata.csv .
Usually wav files are PCM 16 bit 16kHz or 22kHz audio files.

**Comment by erogol** [comment, marked as answer]
https://tts.readthedocs.io/en/latest/formatting_your_dataset.html

**Reply by JustAHippo** [reply]
Thanks so much! I can't wait to get started on this project!
